cmd/minishift/cmd/addon/addon_apply_test.go

```diff
@@ -50,7 +52,5 @@ func Test_unknown_name_for_apply_command_returns_error(t *testing.T) {
 	runApplyAddon(nil, []string{testAddOnName})
 
 	actualOut := tee.StdoutBuffer.String()
-	if expectedOut != actualOut {
-		t.Fatalf("Expected output '%s'. Got '%s'.", expectedOut, actualOut)
-	}
+	assert.Equal(t, expectedOut, actualOut)
```

Comment:
You should use the assert.Equal with a custom message as well. This helps locating the problem when an error occurs.

Reply:
Sure, i will add custom messages that gives some info about what we are testing.

Reply:
When are you planning to do that?

Reply:
although these assertion functions take a custom message, the assert.Equal() assertion will already spit out a message like 'expected something but got something else' on itself. there are some where we might need to add a custom messages in some tests.

Do you still feel like we need custom messages for all?

Reply:
I also wonder how much customization is needed. I expect the test function to fail, and error with "Expected ... but got ...", and this seems to be given. In that case, enough information is available about the failure and to locate where it is.